repo/packages/J/jenkins/24/resource.json

```diff
@@ -7,7 +7,7 @@
   "assets": {
     "container": {
       "docker": {
-        "jenkins-340-2892": "mesosphere/jenkins:3.4.0-2.89.2"
+        "jenkins-341-2892": "mesosphere/jenkins:3.4.1-2.89.2"
```

Comment:
@falseperfection This tag is not found at https://hub.docker.com/r/mesosphere/jenkins/tags/